src/main/java/robot/subsystems/drivetrain/pure_pursuit/PurePursue.java

```diff
@@ -257,4 +257,24 @@ public double getLeftSpeedVoltage(Path path) {
                 kP * (closestPoint().getSpeed() - drivetrain.getLeftSpeed());
     }
 
+    /**
+     * Calculate the change in speed for the robot
+     *
+     * @param currentAcceleration the current acceleration of the robot
+     * @param massCenter the position of the center of mass
+     * @param rollAxis the position of the roll axis
+     * @author Orel
+     */
+    public double fallControl(double currentAcceleration, Point massCenter, Point rollAxis) {
+        double xDifference = massCenter.x - rollAxis.x;
+        double yDifference = massCenter.y - rollAxis.y;
+        double accelerationMax = (xDifference * Constants.g) / yDifference;
```

Comment:
1) `maxAcceleration` is better.
2) What's the meaning of this math? It looks like `atan(robot's angle relative to y axis) * g`, but I am probably wrong.

Reply:
1.fixed
2. this is the calculation of the acceleration that will take down the robot